hack/cleane2e/cleane2e.go

```diff
@@ -7,15 +7,33 @@ import (
 	"context"
 	"os"
 	"sort"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure/auth"
 	"github.com/sirupsen/logrus"
 
 	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/features"
-	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/network"
 	utillog "github.com/Azure/ARO-RP/pkg/util/log"
+
+	"github.com/Azure/ARO-RP/pkg/util/purge"
+	mgmtfeatures "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-07-01/features"
+)
+
+var (
+	subscriptionID string
```

Comment:
Dont initiate those here. Just use `subscription := func` in the code. Anything we expose as a global function or global variable should have good reason to do so.